pkg/vmsock/conn_linux.go

```diff
@@ -88,13 +88,19 @@ func handleUDP(uConn *net.UDPConn, addr *net.UDPAddr, b []byte) {
 		return
 	}
 	data := make([]byte, UDPMaxBufer)
```

Comment:
```suggestion
	data := make([]byte, UDPMaxBuffer)
```
We totally missed the typo…